Sources/TuistLoader/Models+ManifestMappers/TargetAction+ManifestMapper.swift

```diff
@@ -15,10 +15,10 @@ extension TuistGraph.TargetScript {
     {
         let name = manifest.name
         let order = TuistGraph.TargetScript.Order.from(manifest: manifest.order)
-        let inputPaths = try absolutePaths(for: manifest.inputPaths, generatorPaths: generatorPaths)
-        let inputFileListPaths = try absolutePaths(for: manifest.inputFileListPaths, generatorPaths: generatorPaths)
-        let outputPaths = try absolutePaths(for: manifest.outputPaths, generatorPaths: generatorPaths)
-        let outputFileListPaths = try absolutePaths(for: manifest.outputFileListPaths, generatorPaths: generatorPaths)
+        let inputPaths = manifest.inputPaths.map(\.pathString)
+        let inputFileListPaths = manifest.inputFileListPaths.map(\.pathString)
+        let outputPaths = manifest.outputPaths.map(\.pathString)
+        let outputFileListPaths = manifest.outputFileListPaths.map(\.pathString)
```

Comment:
this sadly breaks existing behaviour where the input paths coming from the manifests accept glob patterns which need to be resolved at this point.

see the sample fixture for an example 
https://github.com/tuist/tuist/tree/main/projects/tuist/fixtures/ios_app_with_actions

With those changes we get the incorrect paths in the generated xcodeproj 😞

```diff
     inputPaths = (
-      Sources/AppDelegate.swift,
+      "Sources/**/*.swift",
     );
```